### PR TITLE
RavenDB-18381 - Remove Stats of existing Time Series

### PR DESCRIPTION
**Pull request by shiranshalom**
### Issue link

https://issues.hibernatingrhinos.com/issue/RavenDB-18381

### Type of change

- Bug fix

### How risky is the change?

- Low 

### Backward compatibility

- Not relevant

### Is it platform specific issue?

- No

### Documentation update

- No documentation update is needed 

### Testing 

- Tests have been added that prove the fix is effective or that the feature works

### Is there any existing behavior change of other features due to this change?

- No

### UI work

- No UI work is needed

### CODE REVIEW

File: test/SlowTests/Server/Documents/TimeSeries/TimeSeriesTombstoneCleaner.cs [hunk 1]
```diff
@@ -358,6 +359,94 @@ public async Task CleanTimeSeriesTombstonesInTheClusterWithOnlyFullBackup()
...
+                }
+
+                await EnsureReplicatingAsync(store1, store2);
+
+                var tss = storage.DocumentsStorage.TimeSeriesStorage;
+
+                using (storage.DocumentsStorage.ContextPool.AllocateOperationContext(out DocumentsOperationContext ctx))
+                using (ctx.OpenReadTransaction())
+                {
+                    Assert.True(tss.Stats.GetNumberOfEntries(ctx) >= 6);
```
Comment (karmeli87) [inline test/SlowTests/Server/Documents/TimeSeries/TimeSeriesTombstoneCleaner.cs]: why >=6? this should be deterministic no?
also can we run the cleaner and assert for 0 ?

**Review by karmeli87 — APPROVED** [review]
Please elaborate on the issue & root cause in the PR description

**Comment by karmeli87** [comment]
This PR is only to prevent from the issue to occur. 
We will have a different PR that will handle repairing existing nodes that suffer from this issue.

File: test/SlowTests/Server/Documents/TimeSeries/TimeSeriesTombstoneCleaner.cs (hunk 1 above)
Comment (shiranshalom) [reply test/SlowTests/Server/Documents/TimeSeries/TimeSeriesTombstoneCleaner.cs]: done

**Comment by ppekrol** [comment]
test this please

**Pull request merged**